lib/analysers/taxons/grid_and_leaf_link_analyser.rb

```diff
@@ -11,10 +11,13 @@ def grid_results(taxon)
         taxon.body_html.css(CssSelector.for(:grid_taxon_links))
           .reduce([]) do |results, link|
 
+          links_total = link.ancestors(CssSelector.for(:grid_subsection))
+            .css(CssSelector.for(:grid_taxon_links)).count
+
           results << {
             taxon_base_path: taxon.base_path,
             link_href: link.attr('href'),
-            total_number_of_links: "N/A",
+            total_number_of_links: links_total,
             total_number_of_links_per_section: 'N/A',
```

Comment:
I find this a slightly surprising thing to do; wouldn't it be better to just omit this property from the hash, and deal with the missing property when displaying the results?